@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -708,12 +708,12 @@ export interface SolidLineTechniqueParams extends BaseTechniqueParams, Polygonal
      * For transparent lines, set a value between `0.0` for fully transparent, to `1.0` for fully
      * opaque.
      */
-    opacity?: number;
+    opacity?: MaybeInterpolatedProperty<number>;
     // TODO: Make pixel units default.
     /**
      * Units in which different size properties are specified. Either `Meter` (default) or `Pixel`.
      */
-    metricUnit?: string;
+    metricUnit?: MaybeInterpolatedProperty<string>;
```

Comment:
This shouldn't be a MaybeInterpolatedProperty. We cannot interpolate between `Meter` and `Pixel`.

Reply:
Why not? We can't do it linearly or in cubic now, but discrete interpolation can be done simply.
For example we can say that some road will be 15 meters wide at level 14 or higher, or 8 pixels on lower levels.

Reply:
We can't because right now string interpolation is restricted to color values only.
So this should be implemented in another ticket.